docs/limitations.md

```diff
@@ -58,6 +48,10 @@ configuration. It should be possible to pass this information through to
 the QEMU command line CPU configuration options to gain a similar
 effect.
 
+Note however that the `--cpu-quota` and `--cpu-period` `docker run` options are
```

Comment:
Lets remove some verbiage and have the note written as a directive/admonition:
Note
The `--cpu-quota` ...

Reply:
Ah - I see what you mean ;) I've removed the "however" which makes it slightly more terse, but _note that the_ [*] doc uses the phrase "_note that the_" throughout for consistency.

I'm happy to change all occurrences of that phrase on another commit? It's a shame that github flavoured markdown doesn't support rST goodness like http://docutils.sourceforge.net/docs/ref/rst/directives.html#specific-admonitions :-(

___
[*] pun intended :smile:

Reply:
I approved as after looking through the whole doc, there are just too many places "note" occurs to have them all looking like directives. It will be a lot replacement phrases and a bigger effort than you may want to engage in standing in the way of this PR's merge.